tests/aws/services/cloudwatch/test_cloudwatch.py

```diff
@@ -1393,6 +1393,37 @@ def contains_receive_delete_metrics() -> int:
 
         snapshot.match("get_metric_data_2", response)
 
+    @markers.aws.validated
+    @pytest.mark.skipif(
+        condition=not is_aws_cloud(), reason="Old provider is not raising exception"
+    )
```

Comment:
q: should this test still run for the new provider? is it missing `condition not is_aws_cloud() and is_old_provider()`?

Reply:
oops, copy-pasta 😅 great catch, thanks ❤️